client/src/views/SettingsPage.vue

```diff
@@ -2,49 +2,67 @@
   <div id="changeGameOptions">
     <div class="background"></div>
     <h1>Change Game Settings</h1>
-    <div id="gameOptionsScene"></div>
+    <div v-if="settings.isLoading">
+      Loading...
+    </div>
+    <form id="settings" class="settings" v-if="!settings.isLoading">
+      <setting-line v-for="setting in settings.data" :setting="setting"></setting-line>
+    </form>
+    <button class="button" id="backButton" @click="backClicked">Back</button>
   </div>
 </template>
 
 <script setup lang="ts">
-</script>
+import { useRouter } from "vue-router";
+import SettingLine from "@/components/settings/SettingLine.vue";
+import {useWebsocketHandlers} from "@/utils/useWebsocket";
+import {WebSocketHandler} from "@/ws";
+import {reactive} from "@vue/reactivity";
+import type {GameOptions} from "../../../server/types/GameOptions";
+import type {Setting} from "@/components/settings/settings.types";
 
-<style scoped>
-#changeGameOptions {
-  flex-direction: column;
-  justify-content: center;
-  align-items: center;
-}
+const router = useRouter();
 
-#changeGameOptions {
-  flex-direction: column;
-  justify-content: center;
-  align-items: center;
-  font-size: 20px;
-  color: var(--base-color-fg);
-}
+const settings: {
+  isLoading: boolean,
+  data: {} | GameOptions,
+} = reactive({
+  isLoading: true,
+  data: {},
+});
 
-#changeGameOptions .name {
-  margin-top: 3vw;
-  margin-bottom: 2vw;
-  font-size: 3vw;
+function backClicked() {
+  // TODO : Get the form values, format it and send it to the WS.
```

Comment:
J'ai essayé de lire le contenu du form actuel avec ça : 

```suggestion
  const formData = new FormData(document.querySelector('#settings') as HTMLFormElement);
  console.log('formData', formData);
```

Mais malheureusement y'a rien, ça log `undefined`. J'imagine que c'est pas la bonne pratique et que je suis sensé faire autrement.